src/index.template.html

```diff
@@ -7,6 +7,7 @@
     <link type="text/css" rel="stylesheet" href="./js/vendor/select2/select2.css" />
     <link type="text/css" rel="stylesheet" href="./css/main.css?" />
     <link type="text/css" rel="stylesheet" href="./css/vendor/bootstrap.css" />
+    <link rel="stylesheet" type="text/css" href="./css/dark-theme.css">
```

Comment:
You should switch `type` and `rel` for consistency with the other style sheets.

You might be able to reduce FOIT by just setting the theme class on the `<html>` element (which you can do while the page is loading, although that doesn't help you with setting the button label) and then use descendant selectors for all of your dark mode overrides.

I know this isn't how Pokémon Showdown! does it, but the faster way of doing this is to use a disabled stylesheet; add `disabled="true"` here and also give this element an `id`, and then if the user wants dark mode you can toggle the disabled property of the link. Again, this should also work while the page is still loading to reduce FOIT. (It also avoids needing a separate variable to track whether the user prefers dark mode). You'll also need to remove the dark-theme class on the styles, since they'll be activated automatically depending on whether the stylesheet is disabled. This feature does require IE 9 or later, if that's relevant.

If the content delivery system used by the damage calculator is capable of doing it then using cookies instead of local storage would allow it to send the correct attribute (either `<html class="dark-theme">` or `<stylesheet disabled="true">` as appropriate).